storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2312,6 +2312,12 @@ static bool innodb_init(bool init_dd, bool for_apply_log) {
                                                      : incremental_last_lsn;
   }
 
+  /* backup taken prior to 8.0.23 can have checkoint lsn less more than last
+   * lsn. if so use checkpoint_lsn instead of last_lsn */
```

Comment:
what is less more than? :wink: 
Please fix

Reply:
fixed